doc/classes/DisplayServer.xml

```diff
@@ -233,6 +233,7 @@
 			<return type="int" />
 			<description>
 				Returns the index of the screen containing the window with the keyboard focus, or the primary screen if there's no focused window.
+				[b]Note:[/b] This method is implemented on Linux/X11, macOS, and Windows, on other platforms this method always returns the primary screen.
```

Comment:
There's a lot of cases like these. I wouldn't want to repeat myself to reduce suggestion noise.
```suggestion
				[b]Note:[/b] This method is implemented on Linux/X11, macOS, and Windows. On other platforms, this method always returns the primary screen.
```